devel/altanalyze/altanalyze_dbfetch.cwl

```diff
@@ -0,0 +1,61 @@
+cwlVersion: v1.0
+class: CommandLineTool
+
+requirements: []
+
+hints: []
+
+inputs: 
+  bash_script:
+    type: string?
+    default: |
+      #!/bin/bash
+      python /opt/altanalyze/AltAnalyze.py --species Mm --update Official --version EnsMart72 --additional all
```

Comment:
I think `--species` parameter can be specified in the inputs section. You can use either string of enum type for that. Enum is more preferable as you'll be able to limit the number of option. As far as I remember `--additional all` can be removed.